packages/valory/skills/abstract_round_abci/behaviour_utils.py

```diff
@@ -1795,6 +1714,130 @@ def reset_tendermint_with_wait(
         return True
 
 
+class TmManager(BaseBehaviour, ABC):
```

Comment:
TmManager inherits from BaseBehaviour because it was easier to do so, not because it semantically makes sense. We could get rid of the inheritances given that we can access `BaseBehaviour.reset_tendermint_with_wait()` in a different way.

Reply:
Also more importantly, the rest of the utils related to making requests etc.